specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/stable/2021-09-01/DiskMigrationJobs.json

```diff
@@ -0,0 +1,494 @@
+      "properties": {
+        "error": {
+          "description": "Error definition.",
+          "$ref": "#/definitions/ErrorDefinition"
+        }
+      }
+    },
+    "ErrorDefinition": {
+      "description": "Error definition.",
+      "readOnly": true,
+      "required": [
+        "message",
+        "code"
+      ],
+      "properties": {
+        "code": {
+          "description": "Service specific error code which serves as the substatus for the HTTP error code.",
+          "type": "string",
+          "minimum": 100,
+          "maximum": 600
+        },
+        "message": {
+          "description": "Description of the error.",
+          "type": "string"
+        }
+      }
+    },
+    "MigrationJobStatus": {
+      "description": "Migration job status.",
+      "type": "string",
+      "enum": [
+        "Undefined",
+        "Running",
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Pending"
+      ],
+      "x-ms-enum": {
+        "name": "MigrationJobStatus",
+        "modelAsString": true
+      }
+    },
+    "MigrationSubTaskStatus": {
+      "description": "Migration child task status.",
+      "type": "string",
+      "enum": [
+        "Undefined",
+        "Running",
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Pending",
+        "Skipped"
+      ],
+      "x-ms-enum": {
+        "name": "MigrationSubTaskStatus",
+        "modelAsString": true
+      }
+    },
+    "MigrationSubTaskProperties": {
+      "description": "Disk migration child task properties.",
+      "type": "object",
+      "properties": {
+        "migrationSubtaskStatus": {
+          "description": "The disk migration child task status.",
+          "$ref": "DiskMigrationJobs.json#/definitions/MigrationSubTaskStatus",
+          "readOnly": true
+        },
+        "reason": {
+          "description": "The reason of task failure.",
+          "type": "string",
+          "readOnly": true
+        },
+        "startTime": {
+          "description": "The task start time.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "endTime": {
+          "description": "The task end time.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "targetShare": {
+          "description": "The target share of migration task.",
+          "type": "string",
+          "readOnly": true
+        },
+        "sourceShare": {
+          "description": "The source share of migration task.",
+          "type": "string",
+          "readOnly": true
+        },
+        "targetDiskStateForMigration": {
+          "description": "The disk status.",
+          "$ref": "Disks.json#/definitions/DiskState",
+          "readOnly": true
+        },
+        "diskId": {
+          "description": "The id of disk.",
+          "type": "string",
+          "readOnly": true
+        },
+        "progress": {
```

Comment:
>progress [](http://example.com/codeflow?start=9&length=8)

It would be good to name this according to how progress is measured and reported (e.g. completionPercent, filesMigrated, whatever).

Reply:
Hi, Chris. Thanks for your feedback. Our product code has been checked in so I would prefer not to check the property name. I have changed description for the field. Are you OK with that?

Reply:
OK, I won't block ARM signoff on that.